docroot/modules/custom/webform_template/config/install/webform_template.settings.yml

```diff
@@ -1 +1 @@
+webform_template_elements: "prefix_title:\n  '#type': select\n  '#title': Prefix/Title\n  '#options': titles\nfirst_name:\n  '#type': textfield\n  '#title': 'First Name'\n  '#required': true\nmiddle_initial_middle_name:\n  '#type': textfield\n  '#title': 'Middle Initial/Middle Name'\nlast_name:\n  '#type': textfield\n  '#title': 'Last Name'\n  '#required': true\nsuffix:\n  '#type': select\n  '#title': Suffix\n  '#options':\n    Esq: Esq\n    MD: MD\n    PHD: PHD\n    Jr: Jr\n    Sr: Sr\n    I: I\n    II: II\n    III: III\n    IV: IV\ncompany_organization:\n  '#type': textfield\n  '#title': Company/Organization\nemail:\n  '#type': email\n  '#title': Email\n  '#description': 'We collect this information in accordance with Regulation 1.3.5s7 and <a href=\"https://en.wikipedia.org/wiki/Email\" target=\"_blank\">here&#39;s showing how a link works</a>.'\n  '#required': true\nphone_number:\n  '#type': tel\n  '#title': 'Phone Number'\nfax_number:\n  '#type': tel\n  '#title': 'Fax Number'\nmailing_address_line_1:\n  '#type': textfield\n  '#title': 'Mailing Address Line 1'\nmailing_address_line_2:\n  '#type': textfield\n  '#title': 'Mailing Address Line 2'\ncity:\n  '#type': textfield\n  '#title': City\ncountry:\n  '#type': select\n  '#title': Country\n  '#options': country_names\nstate_province:\n  '#type': select\n  '#title': State/Province\n  '#options': state_province_names\nzip_postal_code:\n  '#type': textfield\n  '#title': 'Zip/Postal Code'\nprocessing_fees:\n  '#type': textfield\n  '#title': 'Processing Fees'\n  '#pattern': '^(?=.)([+-]?([0-9]*)(\\.([0-9]+))?)$'\n  '#counter_type': word\n  '#counter_maximum': '1'\ndelivery_method:\n  '#type': select\n  '#title': 'Delivery Method'\n  '#options':\n    paper: Paper\n    electronic: Electronic\nrequest_category:\n  '#type': select\n  '#title': 'Request Category'\n  '#options':\n    individual: 'Individual (non comercial)'\n    'news media': 'Representative of the news media'\n    educational: Educational\n    commercial: 'Commercial requester'\nrequest_description:\n  '#type': textarea\n  '#title': 'Describe the information you''re requesting'\n  '#required': true\nrequest_fee_waiver:\n  '#type': select\n  '#title': 'Request Fee Waiver'\n  '#options':\n    'yes': 'Yes'\n    'no': 'No'\n  '#required': true\nrequest_expedited_processing:\n  '#type': select\n  '#title': 'Request Expedited Processing'\n  '#options':\n    'yes': 'Yes'\n    'no': 'No'\n  '#required': true\nattachments_supporting_documentation:\n  '#type': managed_file\n  '#title': 'Attachments/Supporting Documentation'\ni_agree_to_the_privacy_policy:\n  '#type': checkbox\n  '#title': 'I agree to the privacy policy'\n  '#required': true\n"
```

Comment:
A few items:
1. I noticed the state dropdown isn't dependent on the selected country. Would it be a large lift to get that functionality?
1. Typo: `non comercial`: "comercial" should be "commercial"
1. Looking at the excel spreadsheet, it seems like the expectation for the `Request Fee Waver` and `Request Expedited Processing` fields should have text fields appear if 'Yes' is selected to allow the user to enter an explanation.
1. Mind if we remove the "I agree to the privacy policy" checkbox?
1. Mind if we remove the text for the Email field that says "We collect this information in accordance with Regulation 1.3.5s7 and here's showing how a link works."